bin/s2nc.c

```diff
@@ -256,11 +262,12 @@ int main(int argc, char *const *argv)
         {"timeout", required_argument, 0, 't'},
         {"corked-io", no_argument, 0, 'C'},
         {"tls13", no_argument, 0, '3'},
+        {"curves", required_argument, NULL, 'u'},
     };
 
     while (1) {
         int option_index = 0;
-        int c = getopt_long(argc, argv, "a:c:ehn:sf:d:D:t:irTC", long_options, &option_index);
+        int c = getopt_long(argc, argv, "a:c:ehn:sf:d:D:t:irTCu", long_options, &option_index);
```

Comment:
Have you tested with your "-u" argument? I believe you need "u:" to indicate that it takes an argument.